traffic_portal/package.json

```diff
@@ -30,5 +30,8 @@
     "requirejs": "2.3.6",
     "ssl-root-cas": "1.3.1",
     "time-grunt": "1.3.0"
+  },
+  "dependencies": {
+    "ag-grid-community": "^23.2.0"
```

Comment:
i guess i'm less risk tolerant but i'd do `23.2.0` or `~23.2.0` rather than letting it roll ahead to new minor versions, but your call.

Reply:
That's just the default behavior when you `npm install --save` something.

I don't see a problem with letting it roll ahead to new minor versions; the API won't ever break so it should only come with more benefits. Besides, pinning these things and never coming back is how we wind up on unsupported releases for years.